[2022-07-14 00:00:08,853] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2022-07-13T00:00:00+00:00 [queued]>
[2022-07-14 00:00:08,864] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2022-07-13T00:00:00+00:00 [queued]>
[2022-07-14 00:00:08,864] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-14 00:00:08,864] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-14 00:00:08,865] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-14 00:00:08,881] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): store_user> on 2022-07-13 00:00:00+00:00
[2022-07-14 00:00:08,886] {standard_task_runner.py:52} INFO - Started process 19180 to run task
[2022-07-14 00:00:08,890] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'scheduled__2022-07-13T00:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpxlr2juuo', '--error-file', '/tmp/tmp_l4sas3_']
[2022-07-14 00:00:08,892] {standard_task_runner.py:80} INFO - Job 30: Subtask store_user
[2022-07-14 00:00:08,961] {task_command.py:371} INFO - Running <TaskInstance: user_processing.store_user scheduled__2022-07-13T00:00:00+00:00 [running]> on host b0c7b58a654d
[2022-07-14 00:00:09,041] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2022-07-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-13T00:00:00+00:00
[2022-07-14 00:00:09,043] {postgres.py:127} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2022-07-14 00:00:09,051] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-07-14 00:00:09,057] {python.py:173} INFO - Done. Returned value was: None
[2022-07-14 00:00:09,071] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20220713T000000, start_date=20220714T000008, end_date=20220714T000009
[2022-07-14 00:00:09,102] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-14 00:00:09,137] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
